clarity.d.ts

```diff
@@ -81,6 +81,16 @@ declare const enum State {
   Unloading
 }
 
+declare const enum EventType {
+  /* 0 */ Discover,
+  /* 1 */ Instrumentation,
```

Comment:
This is a breaking change for data consumption. Is the goal to save space?

Reply:
It is, partially, but also makes the code more neat. There are multiple other ways to reduce payloads and most of them will be breaking changes too. Let's sync offline about how to move forward with those. I won't check in this PR for now.

---
In reply to: [171736011](https://github.com/Microsoft/clarity-js/pull/70#discussion_r171736011) [](ancestors = 171736011)